cmd/processor/app/processor.go

```diff
@@ -129,12 +129,14 @@ func NewProcessor(configurationPath string, platformConfigurationPath string) (*
 	newProcessor.logger.DebugWith("Read configuration",
 		"config", string(indentedProcessorConfiguration))
 
-	// restore function configuration from secret
-	restoredFunctionConfig, err := newProcessor.restoreFunctionConfig(&processorConfiguration.Config)
-	if err != nil {
-		return nil, errors.Wrap(err, "Failed to restore function configuration")
+	// restore function configuration from secret if needed
+	if restoreConfigFromSecret := common.GetEnvOrDefaultBool("NUCLIO_RESTORE_FUNCTION_CONFIG_FROM_SECRET", false); restoreConfigFromSecret {
```

Comment:
will that be set if platform/function indicate "skip function secret scrubbing"?